pkg/tcpip/stack/bridge_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package bridge_test
+package stack_test
```

Comment:
it isn't a part of stack_test...

Reply:
@milantracy could you take a look whether we can run these tests as part of stack_test?